test/test-index-with-dir-env.js

```diff
@@ -35,30 +36,23 @@ test.cb('CUMULUS_MESSAGE_ADAPTER_DIR sets the location of the message adapter',
   const expectedOutput = clonedeep(testContext.outputEvent);
   expectedOutput.payload = businessLogicOutput;
 
-  function callback(err, data) {
-    t.is(err, null);
-    t.deepEqual(data, expectedOutput);
-    t.end();
-  }
-
-  return cumulusMessageAdapter.runCumulusTask(businessLogic, testContext.inputEvent, {}, callback);
+  const result = await cumulusMessageAdapter.runCumulusTask(
+    businessLogic,
+    testContext.inputEvent,
+    {}
+  );
+  t.deepEqual(result, expectedOutput);
 });
 
-test.cb('callback returns error if CUMULUS_MESSAGE_ADAPTER_DIR is incorrect', (t) => {
+test('callback returns error if CUMULUS_MESSAGE_ADAPTER_DIR is incorrect', async(t) => {
   const dir = path.join(__dirname, 'wrong-dir', 'cumulus-message-adapter');
   process.env.CUMULUS_MESSAGE_ADAPTER_DIR = dir;
 
   const businessLogicOutput = 42;
-  const businessLogic = () => businessLogicOutput;
+  const businessLogic = async () => businessLogicOutput;
 
   const inputEvent = { a: 1 };
 
-  function callback(err, data) {
-    t.is(typeof err, 'object');
-    t.is(err.name, 'CumulusMessageAdapterExecutionError');
-    t.is(data, undefined);
-    t.end();
-  }
-
-  cumulusMessageAdapter.runCumulusTask(businessLogic, inputEvent, {}, callback);
+  await t.throwsAsync(cumulusMessageAdapter.runCumulusTask(businessLogic,
```

Comment:
Does this mean that the thrown error is null?

Reply:
No, this is completely irrelevant/an extra param.  Good catch.  9a15062